reproducibility_project/src/engines/cassandra/project.py

```diff
@@ -17,6 +17,255 @@ def __init__(self):
         self.ff_fn = self.data_dir / "forcefield.xml"
 
 
+@Project.label
+def is_cassandra(job):
+    return job.sp.simulation_engine == "cassandra"
+
+
+@Project.operation
```

Comment:
This will need either the is_cassandra call above for each call, or as a lambda for each operation
```
@Project.operation
@Project.pre(lambda j: j.sp.engine == "cassandra")
@Project.post(cassandra_complete)
@directives(omp_num_threads=4)